client/jetpack-cloud/sections/sidebar-navigation/manage-selected-site.tsx

```diff
@@ -118,6 +122,15 @@ const useMenuItems = ( {
 					enabled: isAdmin && ! isWPCOM && ! isWPForTeamsSite,
 					isSelected: itemLinkMatches( path, scanPath( siteSlug ) ),
 				},
+				{
```

Comment:
Nit: Might be good to add a comment to this particular entry to differentiate between the difference between the one above where both has the title 'Scan'